### PR TITLE
add citation and descriptions for the data sources

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -57,17 +60,24 @@ def main():
         elif args.command == 'citation':
             if args.dataset is None:
                 citation_path = os.path.join(os.path.split(__file__)[0], '../CITATION')
-                print citation_path
+                print "\ncitation for retriever\n"
```
Comment (ethanwhite) [inline __main__.py]: "citation for retriever\" -> "Citation for retriever:"

File: __main__.py [hunk 2]
```diff
@@ -57,17 +60,24 @@ def main():
...
                 with open(citation_path) as citation_file:
                     print citation_file.read()
             else:
                 scripts = name_matches(script_list, args.dataset)
                 for dataset in scripts:
-                    print dataset.description
+
+                    print "\ncitation::   %s" %dataset.citation 
+                    print "\ndescription::    %s\n" %dataset.description
```
Comment (ethanwhite) [inline __main__.py]: Capitalize 'citation' and 'description', and move to using `.format` syntax instead of `%`.

File: lib/templates.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ class Script:
     """This class represents a database toolkit script. Scripts should inherit
     from this class and execute their code in the download method."""
     def __init__(self, name="", description="", shortname="", urls=dict(),
-                 tables=dict(), ref="", public=True, addendum=None, **kwargs):
+                 tables=dict(), ref="", public=True, addendum=None, citaion="No Defined",**kwargs):
```
Comment (ethanwhite) [inline lib/templates.py]: "citaion" -> "citation"
"No Defined" -> "Not currently available"

File: lib/templates.py [hunk 4]
```diff
@@ -21,6 +21,7 @@ def __init__(self, name="", description="", shortname="", urls=dict(),
         self.ref = ref
         self.public = public
         self.addendum = addendum
+        self.citation= citaion
```
Comment (ethanwhite) [inline lib/templates.py]: "citaion" -> "citation"

File: scripts/EA_Matter_2014.script [hunk 5]
```diff
@@ -1,6 +1,7 @@
 name: Spatial Population Data Alpine Butterfly -- Matter et al 2014
-description: Stephen F. Matter, Nusha Keyghobadhi, and Jens Roland. 2014. Ten years of abundance data within a spatial population network of the alpine butterfly, Parnassius smintheus. Ecology 95:2985
 shortname: Matter2014
+description: Stephen F. Matter, Nusha Keyghobadhi, and Jens Roland. 2014. Ten years of abundance data within a spatial population network of the alpine butterfly, Parnassius smintheus. Ecology 95:2985
```
Comment (ethanwhite) [inline scripts/EA_Matter_2014.script]: Remove title, authors, etc. from `description`.

File: scripts/EA_Rodriguez-Buritica2013.script [hunk 6]
```diff
@@ -1,5 +1,6 @@
 name: Sonoran Desert Perennials
-description: Susana Rodriguez-Buritica, Helen Raichle, Robert H. Webb, Raymond M. Turner, and D. Lawrence Venable. 2013. One hundred and six years of population and community dynamics of Sonoran Desert Laboratory perennials. Ecology 94:976.
+citation: data set constitutes all information associated with the Spalding-Shreve permanent vegetation plots from 1906 through 2012, which is the longest-running plant monitoring program in the world
```
Comment (ethanwhite) [inline scripts/EA_Rodriguez-Buritica2013.script]: I'm guessing this should be `description` not `citation`.

File: scripts/EA_Turcotte.script [hunk 7]
```diff
@@ -1,4 +1,6 @@
 name: Leaf herbivory dataset
-description: Turcotte et al. 2014. Percentage leaf herbivory across vascular plant species. Ecology 95:788
+citation: spatially explicit measurements of population level leaf herbivory on 1145 species of vascular plants from 189 studies from across the globe
```
Comment (ethanwhite) [inline scripts/EA_Turcotte.script]: `description` not `citation`?

File: scripts/EA_Wilman_2014.script [hunk 8]
```diff
@@ -1,7 +1,10 @@
 name: Mammal and Bird foraging attributes -- Wilman, et al., 2014
-description: Hamish Wilman, Jonathan Belmaker, Jennifer Simpson, Carolina de la Rosa, Marcelo M. Rivadeneira, and Walter Jetz. 2014. EltonTraits 1.0: Species-level foraging attributes of the world's birds and mammals. Ecology 95:2027. 
+
+description: characterization of Species by physiological, behavioral, and ecological attributes that are subjected to varying evolutionary and ecological constraints and jointly determine their role and function in ecosystems.
+citation: Hamish Wilman, Jonathan Belmaker, Jennifer Simpson, Carolina de la Rosa, Marcelo M. Rivadeneira, and Walter Jetz. 2014. EltonTraits 1.0: Species-level foraging attributes of the world's birds and mammals. Ecology 95:2027. 
 shortname: EltonTraits
 tags: Taxon > Mammals, Taxon > Birds, Data Type > Compilation
+citation
```
Comment (ethanwhite) [inline scripts/EA_Wilman_2014.script]: Delete

**Comment by ethanwhite** [comment]
Looks good overall. Thanks for going to the extra effort of tracking things down and adding descriptions in a number of cases. I've made a few in-line comments. Two more general suggestions:

1. In a number of cases you've changed the label on the existing `description` field to `citation`. That's great, but it would be good to keep a description as well, so I'd recommend doing what you've already done in a couple of cases which is to have the current `citation` field like you've already done and then add a `description` field with just the title.
2. The `description` field should use sentence style capitalization and so always start with a capital letter.

**Comment by ethanwhite** [comment]
Note to self: This needs to be tested again once the changes are in since none of the citations are currently showing, probably due to the `citaion` typo.

**Comment by ethanwhite** [comment]
Merged in 8768066faf9b859fbbd7b81719989666012a8e99.
